tcms/testplans/templates/testplans/get.html

```diff
@@ -84,12 +83,16 @@ <h2 class="card-pf-title">
         <div class="col-xs-12 col-sm-12 col-md-12">
             <div class="card-pf card-pf-accented">
                 <div class="card-pf-heading">
-                    <h2 class="card-pf-title">
+                    <div class="card-pf-title">
                         {% trans 'Confirmed test cases' %}
-                    </h2>
+                    </div>
```

Comment:
This is a h2 anywhere else. Why change ?

Reply:
I try to follow the examples from patternfly. Also the spacing is better I guess.

Reply:
IMO keep it consistent with what we already have. It is possible that Patternfly examples have also been updated meanwhile and we're not ready to upgrade just yet.